packages/subgraph/config/local.json

```
@@ -1,3 +1,4 @@
 {
-  "local": true
+  "local": true,
+  "owner": "0x0000000000000000000000000000000000000000"
```

Could we use the default hardhat account in this case? 
It seems to be a consistent address across our environments.

```suggestion
  "owner": "0x90f8bf6a479f320ead074411a4b0e7944ea8c9c1"
```

---

I see that this is fixed in 8ec6dfe8cbc53a98749ac6f79c53333f70348f06. Thanks!